mne/channels/montage.py

```diff
@@ -72,6 +74,10 @@ def __repr__(self):
              % (self.kind, len(self.ch_names), ', '.join(self.ch_names[:3])))
         return s
 
+    @property
+    def pos2d(self):
```

Comment:
I thought in this case it would be nice to have both `pos` and `pos2d` attributes, also because the calculation is fast. But if you want I'll change it into a method.